lib/resultuploaders/result_uploader.rb

```diff
@@ -10,24 +11,49 @@ class ResultUploader
     # UploaderFactory
     #
     class UploaderFactory
-      UPLOADERS = {
-        dropbox: Dropbox
-      }.freeze
+      def self.initialize(logger)
+        @uploaders = {}
+
+        Dir["#{__dir__}/*"].each do |dir|
```

Comment:
This is problematic if `__dir__` contains `*` though it's very unlikely. Use `Dir#each_child`.

Reply:
Fixed